test/functional/wallet_bumpfee.py

```diff
@@ -253,12 +253,16 @@ def test_dust_to_fee(self, rbf_node, dest_address):
     self.log.info('Test that bumped output that is dust is dropped to fee')
     rbfid = spend_one_input(rbf_node, dest_address)
     fulltx = rbf_node.getrawtransaction(rbfid, 1)
-    # size of transaction (p2wpkh, 1 input, 2 outputs): 141 vbytes
-    assert_equal(fulltx["vsize"], 141)
-    # bump with fee_rate of 0.00350000 BTC per 1000 vbytes
-    # expected bump fee of 141 vbytes * fee_rate 0.00350000 BTC / 1000 vbytes = 0.00049350 BTC
-    # but dust is dropped, so actual bump fee is 0.00050000
-    bumped_tx = rbf_node.bumpfee(rbfid, {"fee_rate": 0.0035})
+    # The DER formatting used by Bitcoin to serialize ECDSA signatures means signatures can have
+    # a variable size of 70-72 bytes, with most being 71 or 72 bytes. The signature in the witness
+    # is divided by 4 for the vsize, so this variance can take it across a vbyte boundary. Thus
+    # the expected transaction size (p2wpkh, 1 input, 2 outputs) is 140-141 vbytes -- usually 141.
+    assert 140 <= fulltx["vsize"] <= 141
```

Comment:
If this fails, we don't get any useful output (the assert return value is False). If you wanted to be really verbose, you could do something like:

```
if fulltx['vsize'] not in [140, 141]:
    print("Invalid tx size: {}, full tx: {}".format(fulltx['vsize'], fulltx))
    raise AssertionError
```

That's perhaps overkill, but more verbose logging makes debugging intermittent failures much easier.

Reply:
Good point; thanks for the suggestion. Updated (with the x <= y <= z operator as it apparently runs faster?)